cantabular-import/helpers/extract-job-info/extract-job-info.go

```diff
@@ -127,29 +165,48 @@ func main() {
 
 			printAndSave(idResultFile, fmt.Sprintf("   first event time: %s", firstTime))
 			printAndSave(idResultFile, fmt.Sprintf("    last event time: %s", lastTime))
+			if idsFound > 1 {
+				printAndSave(idResultFile, fmt.Sprintf("max id execution time is: %d.%09d seconds, finishing at: %s\n", maxDiff/1000000000, maxDiff%1000000000, maxDiffTime))
```

Comment:
I would suggest using the Seconds() function (implemented by the Duration struct) instead, which does the same

Reply:
Done